libs/common/src/helpers/is-gef-facility-type.ts

```diff
@@ -0,0 +1,9 @@
+import { FACILITY_TYPE } from '../constants';
+import { FacilityType } from '../types';
+
+/**
+ * A facility belongs to a GEF deal if and only if the facility type is 'CASH' or 'CONTINGENT'.
+ *
```

Comment:
can this empty line be removed please?

Reply:
Refactored, this file no longer features in this PR